pkg/icingadb/ha.go

```diff
@@ -121,7 +121,8 @@ func (h *HA) Takeover() chan string {
 
 // State returns the status quo.
 func (h *HA) State() (responsibleTsMilli int64, responsible, otherResponsible bool) {
-	state, _ := h.state.Load()
+	state := h.state.Load()
+
```

Comment:
This has been a zero value if uninitialized, now it's a nil ptr in this case. Now we have to make sure HA#State() don't get a nil ptr.

Reply:
Done.